app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1165,6 +1168,8 @@ class BrowserTabViewModel @Inject constructor(
 
         canAutofillSelectCredentialsDialogCanAutomaticallyShow = true
 
+        onLoadProgressChanged(progress = newWebNavigationState.progress ?: 0)
```

Comment:
this should be in `fun progressChanged()` and checking that progress is 100.

Reply:
Didn’t know about this method. I’ll move it there.